lib/cpp/src/thrift/windows/config.h

```diff
@@ -36,6 +36,9 @@
 #define USE_BOOST_THREAD 1
 #endif
 
+// Something that defines PRId64 is required to build
+#define HAVE_INTTYPES_H 1
+
```

Comment:
Note I have opened THRIFT-4025 because I don't see why we have a custom config.h file on Windows when we go through the hassle of generating one with cmake.